### ISSUE TITLE
reek plugin keeps emitting stack traces

### ISSUE THREAD

**Issue by bronson** [AtomLinter/linter-reek#92]
While I'm editing code, I might temporarily have unbalanced parentheses... When I do, Reek starts hitting me with Atom errors, over and over.

Is there any way to make Reek stop doing this?  Listing the errors at the bottom of the editor window is fine.

<img width="477" alt="screen shot 2016-10-13 at 1 50 44 pm" src="https://cloud.githubusercontent.com/assets/1776/19366484/177a32c6-914c-11e6-9475-a72fdd29f728.png">

**Comment by Arcanemagus** [comment]
That's coming from the `stderr` output of `reek`, does it work for you on the CLI?

**Comment by bronson** [comment]
Yes, it works fine.  When the file doesn't contain a syntax error, everything runs with no output.

When I'm editing the file, reek might output:

```
$ reek --version
reek 4.4.2
$ reek bin/config.rb 
bin/config.rb:5:8: error: unexpected token tDOT
bin/config.rb:5: unless .include?(branch)
bin/config.rb:5:        ^                
bin/config.rb: Parser::SyntaxError: unexpected token tDOT
```

Which is correct, but I'm in the middle of editing the file.

**Comment by Arcanemagus** [comment]
Hmm, maybe `reek` has changed their output recently to show syntax errors differently?

**Comment by bronson** [comment]
Just checked, I don't have it set to lint-as-you-type.

However, I DO have the atom/autosave plugin enabled (which I think is the default?)  So my editors get saved whenever they lose focus, which can be quite often.

**Comment by bronson** [comment]
Sorry, nope.  It's disabled by default.

But I really really like it.  I'd prefer to keep it enabled.  :)

**Comment by dhagerty9009** [comment]
I'm going to work on a way to still alert the user to invalid syntax, but by showing a more severe version of the regular output of the linter.
